src/cartesian/getTicks.ts

```diff
@@ -86,9 +39,7 @@ function getTicksEnd({
       result[i] = entry = { ...entry, tickCoord: entry.coordinate };
     }
 
-    const isShow =
-      sign * (entry.tickCoord - (sign * size) / 2 - start) >= 0 &&
-      sign * (entry.tickCoord + (sign * size) / 2 - end) <= 0;
+    const isShow = doesTickFitInBetweenStartAndEnd(sign, entry.tickCoord, size, start, end);
```

Comment:
nit: i know this was here before but can we call this `isVisible` or something a little easier to read?